lib/rules/jsx-no-bind.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @fileoverview Prevents usage of Function.prototype.bind and arrow functions
+ *               in React component definition.
+ * @author Daniel Lo Nigro <dan.cx>
+ */
+'use strict';
+
+// -----------------------------------------------------------------------------
+// Rule Definition
+// -----------------------------------------------------------------------------
+
+module.exports = function(context) {
+  var configuration = context.options[0] || {};
+
+  return {
+    JSXAttribute: function(node) {
+      if (!node.value || !node.value.expression) {
+        return;
+      }
+      var valueNode = node.value.expression;
+      if (
+        !configuration.allowBind &&
+        valueNode.type === 'CallExpression' &&
+        valueNode.callee.property.name === 'bind'
```

Comment:
This is a really naïve check for bind (it just checks for calls to any method called`.bind) but I figure it's Good Enough™. I hope nobody has objects with methods called `bind` and uses them in this context. I imagine that's an edge case. :stuck_out_tongue: